app/users/delete/template.hbs

```diff
@@ -5,6 +5,6 @@
     updateButtonText=updateButtonText }}
   <div class="alert alert-danger">
     <span class="glyphicon glyphicon-warning-sign"></span>
-    {{t 'messages.areYouSureDelete' user=model.name}}
+    {{t 'messages.delete' name=(concat "the user " model.name)}}
```

Comment:
This code is passing a non localized string when it should be passing in a localized string or it should use the name of the item being deleted.